tsconfig.json

```diff
@@ -1,6 +1,13 @@
 {
   "compilerOptions": {
-    "plugins": [{ "name": "typescript-tslint-plugin" }],
+    "plugins": [
```

Comment:
`typescript-plugin-css-modules` config will be followed up in another PR (it's missing a declaration file and other setup)

Reply:
Decided to remove this to keep the PR clean and single-purpose.